app/views/main/UnknownPage.js

```diff
@@ -49,15 +46,15 @@ export const UnknownPage = () => {
           </View>
           <View style={styles.contentBelowPulse}>
             <Text style={styles.mainTextBelow}>
-              {t('label.home_unknown_header')}
+              {t('label.home_setting_off_header')}
             </Text>
             <Typography style={styles.subheaderText}>
               {t('label.home_unknown_subtext')}
```

Comment:
Separate issue, this doesn't have an attached `.en` string. 
This file should never be hit also.
 
I think ideally we should scrap this file and have pages for:
1. location off (for phone)
2. location permissions disabled for the app